reledpar.dtx

```diff
@@ -4446,7 +4446,7 @@
   \global\advance\edtext@later by \@ne%
 %    \end{macrocode}
 %    \begin{macrocode}
-% As we are in a simili-\cs{edtext}, we need no to get the \cs{sameword} data stored in the auxiliary file for this \cs{edtextlater}.
+% As we are in a simili-\cs{edtext}, we need not get the \cs{sameword} data stored in the auxiliary file for this \cs{edtextlater}. %Please note that I'm not sure what is meant by "simili-".
```

Comment:
> On 2016-07-01, at 03:53 , Maïeul Rouquette <notifications@github.com> wrote:
> 
> In reledpar.dtx <https://github.com/maieul/ledmac/pull/550#discussion_r69262163>:
> 
> > @@ -4446,7 +4446,7 @@
> >    \global\advance\edtext@later by \@ne%
> >  %    \end{macrocode}
> >  %    \begin{macrocode}
> > -% As we are in a simili-\cs{edtext}, we need no to get the \cs{sameword} data stored in the auxiliary file for this \cs{edtextlater}.
> > +% As we are in a simili-\cs{edtext}, we need not get the \cs{sameword} data stored in the auxiliary file for this \cs{edtextlater}. %Please note that I'm not sure what is meant by "simili-".
> simili is a french expression from Latin. Apparently it does not exist on English. "no" was for "now”.
> 
Ah, “imitation” then! In this case, I would write, “As we are in a pseudo-\cs{edtext}, we now need to get . . . “

M





> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/maieul/ledmac/pull/550/files/abc94f75401202c18ccac19e1d798f85234d748f#r69262163>, or mute the thread <https://github.com/notifications/unsubscribe/AHG-QDoVuHbXEKEtDC7WnUOPdjc-Y0oHks5qRMdjgaJpZM4JC5Lv>.
>